updating/installing-update-service.adoc

```diff
@@ -22,7 +22,32 @@ include::modules/update-service-overview.adoc[leveloffset=+1]
 == Prerequisites
 
 * For more information on installing Operators, see xref:../operators/user/olm-installing-operators-in-namespace.adoc#olm-installing-operators-in-namespace[Installing Operators in your namespace].
-* If the release images are contained in a secure registry, complete the steps in xref:../registry/configuring-registry-operator.adoc#images-configuration-cas_configuring-registry-operator[Configuring additional trust stores for image registry access].
+
+[id="update-service-registry-configuration"]
+=== Configuring additional trust stores for image registry access
+
+If the release images are contained in a secure registry, complete the steps in xref:../registry/configuring-registry-operator.adoc#images-configuration-cas_configuring-registry-operator[Configuring additional trust stores for image registry access] along with following changes for the update service.
+
+The update service operator needs the config map key name `updateservice-registry` in the registry CA cert.
```

Comment:
Operator names are treated as proper nouns, so "The Update Service Operator..."

For reference to the guidelines: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#operator-capitalization

Reply:
We've been referring to the operator throughout as `OpenShift Update Service Operator` so use that.

Reply:
Resolved in https://github.com/openshift/openshift-docs/commit/dd0bc6c574ae5b287a43ff8bddc77f8ebc8f1db7